### PR TITLE
Showing the loading state on the CreatorLocks component

### PR DESCRIPTION
**Pull request by julien51**
This is the final PR for #2302 for now.
We show the loading state on the creator dashboard.

Refs #2302

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [X] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [X] My code follows the style guidelines of this project, including naming conventions
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] If my code adds or changes components, I have written corresponding stories with Storybook
- [X] I have performed a self-review of my own code
- [X] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
image

**Comment by julien51** [comment]
image

File: unlock-app/src/components/creator/CreatorLocks.js [hunk 1]
```diff
@@ -60,16 +61,21 @@ export class CreatorLocks extends React.Component {
...
-        {lockFeed.length === 0 && !showDashboardForm && (
+        {lockFeed.length === 0 && !loading && !showDashboardForm && (
           <DefaultError
             title="Create a lock to get started"
             illustration="/static/images/illustrations/lock.svg"
             critical={false}
           >
-            If you have already created some locks, they should appear here
-            momentarily.
+            You have not created any lock yet. Create your first lock in seconds
```
Comment (cnasc) [inline unlock-app/src/components/creator/CreatorLocks.js]:
```suggestion
            You have not created any locks yet. Create your first lock in seconds
```

File: unlock-app/src/components/creator/CreatorLocks.js [hunk 2]
```diff
@@ -60,16 +61,21 @@ export class CreatorLocks extends React.Component {
...
+            by clicking on the &#8216;Create Lock&#8217;.
```
Comment (cnasc) [inline unlock-app/src/components/creator/CreatorLocks.js]:
```suggestion
            by clicking on the &#8216;Create Lock&#8217; button.
```

**Comment by julien51** [comment]
Thanks @cnasc !

**Pull request merged**